Symfony/CS/Fixer/PSR0/Psr0Fixer.php

```diff
@@ -161,6 +160,12 @@ public function supports(\SplFileInfo $file)
             return false;
         }
 
+        $tokens = token_get_all(sprintf('<?php %s', $filenameParts[0]));
```

Comment:
if anyone knows a better way to check if a string is predefined constant or keyword in PHP, I'm happy to update :)